pkg/Microsoft.Private.CoreFx.NETCoreApp/Microsoft.Private.CoreFx.NETCoreApp.builds

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  
+  <Import Project="$(MSBuildProjectName).props" />
+  
+  <ItemGroup>
+    <!-- identity project, runtime specific projects are included by props above -->
+    <Project Include="$(MSBuildProjectName).pkgproj" />
+  </ItemGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.traversal.targets))\dir.traversal.targets" />
+
+  <UsingTask TaskName="GetTargetMachineInfo" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll"/>
+  <Target Name="FilterPackageProjects" AfterTargets="FilterProjects">
+    <!-- If we're not on Windows, and RID wasn't specified, get native RID -->
+    <GetTargetMachineInfo Condition="'$(OS)' != 'Windows_NT' AND '$(TestNugetRuntimeId)' == ''">
```

Comment:
Ok, so how about this: we create a central target "GetNuGetRuntimeIdentifier" that computes the as NuGetRuntimeIdentifier.  We'll set it to TestNugetRuntimeId if that is already set.  The target won't be sequenced since most projects don't need this property set and actually setting it will cause them to behave differently.  That target can live in buildtools.  Any preference on a targets file for it?